addons/refuel/functions/fnc_makeJerryCan.sqf

```diff
@@ -22,12 +22,12 @@ if (isNull _target ||
     {_target isKindOf "AllVehicles"} ||
     {_target getVariable [QGVAR(jerryCan), false]}) exitWith {};
 
-if (isServer) then {
-    [_target, _fuelAmount] call FUNC(setFuel);  // has global effects
-};
 _target setVariable [QGVAR(jerryCan), true];
 _target setVariable [QGVAR(source), _target];
 
+_target setVariable [QGVAR(capacity), _fuelAmount];
+[_target, _fuelAmount] call FUNC(setFuel);
```

Comment:
```suggestion
if (isServer) then { [_target, _fuelAmount] call FUNC(setFuel); };
```
`setFuel` does `_source setVariable [QGVAR(currentFuelCargo), (_fuel min _capacity) max 0, true];` 
so I think anytime someone jips it would refill all jerry cans

Reply:
Ah, good catch!